src/components/Nav.jsx

```diff
@@ -1,12 +1,28 @@
-import './Nav.css';
+import styles from "./Nav.module.css";
+import Nav_logo from "../assets/nav_panda_logo_img.png";
```

Comment:
요소 이름은 파스칼 케이스가 일반적인 거 같아용
```suggestion
import NavLogo from "../assets/nav_panda_logo_img.png";
```